synapse/lib/tufo.py

```diff
@@ -2,6 +2,8 @@
 Some common utility functions for dealing with tufos.
 '''
 
+# FIXME do not delete until all functionality is ported to Node
```

Comment:
I'm going to start putting these FIXMEs at the header of the remaining modules that need to be ported (and to unit test files that are skipped w/ the reason why. Ex: we can't run these because the module doesn't work because we don't have configable)